### DISCUSSION TITLE
Issue with the tablet area tool

### DISCUSSION THREAD

**Discussion by etmclovin** [ppy/osu#27229] [Ideas] [answered]
I am someone who plays with a very specific play area as i feel it gives me the best performance at osu and it is what im used to
My OpenTabletDriver area:
![image](https://github.com/ppy/osu/assets/85808692/9e2c64c4-aea2-4251-8dd0-44d0472c2e28)
however recently ive been forcing myself to use lazer as i think its a much better client and now that you can earn pp i feel now is the time to switch but ive been cursed with just missing circles and i think it is because lazers built in tablet area tool doesnt let me input these very specific values and only lets me use whole numbers these numbers specifically are:
x offset: 45
y offset 25
width: 89
height: 50

even though the difference is minimal it is noticeable and i wish i could do one of two things:
1. have the ability to input these very specific values (and not have to use the slider as its very unintuitive and difficult to use with prescision)
2. have the ability to turn off the lazer tablet area tool without it blocking all tablet input so i can use OpenTabletDriver instead similar to how osu!stable handles its tablet input

i have searched for a similar issue and i did find one (https://github.com/ppy/osu/discussions/13188) but this one does not give the idea ive listed above as 2.

any info about this issue or possible upcoming changes/fixes for it would be appreciated thanks!

**Comment by Stoppedpuma** [comment, marked as answer]
You can keep using OpenTabletDriver externally and just disable the "Tablet" option in-game.

>turn off the lazer tablet area tool without it blocking all tablet input

Try enabling / disabling "High precision mouse"

**Reply by etmclovin** [reply]
Im not home at the moment but i will try that once i am

**Reply by etmclovin** [reply]
yeah that worked, thanks for your help
